projects/plugins/crm/src/automation/commons/conditions/invoices/class-invoice-field-contains.php

```diff
@@ -18,27 +19,36 @@
 class Invoice_Field_Contains extends Base_Condition {
 
 	/**
-	 * All valid operators for this condition.
+	 * Invoice_Field_Contains constructor.
 	 *
 	 * @since $$next-version$$
-	 * @var string[] $valid_operators Valid operators.
-	 */
-	protected $valid_operators = array(
-		'contains',
-		'does_not_contain',
-	);
-
-	/**
-	 * All valid attributes for this condition.
 	 *
-	 * @since $$next-version$$
-	 * @var string[] $valid_operators Valid attributes.
+	 * @param array $step_data The step data.
 	 */
-	private $valid_attributes = array(
-		'field',
-		'operator',
-		'value',
-	);
+	public function __construct( array $step_data ) {
+		parent::__construct( $step_data );
+
+		// TODO: Fetch automation fields from our DAL.
```

Comment:
For this to-do task, we could use the `objectModel`, which has all the information and label translation done. We have to add a method to retrieve the model, `get_object_model()`.

Reply:
This can be done in a later PR, agree?